src/parser/parse_tree.h

```diff
@@ -3665,6 +3665,7 @@ struct parser_node
     unsigned is_system_generated_stmt:1;	/* is internally generated by system */
     unsigned use_auto_commit:1;	/* use autocommit */
     unsigned done_reduce_equality_terms:1;	/* reduce_equality_terms() is already called */
+    unsigned is_false_where:1;	/* reduced node from false where */
```

Comment:
Adding it to pt_value_info looks better.

Reply:
I fixed as you comment.